### ISSUE TITLE
The running results of SWAP_CNN are much lower than the reported results.

### ISSUE THREAD

**Issue by Andrewymd** [pym1024/SWAP_Universal#1]
Thanks a lot for your work, could you help me figure out where the problem is?
For instance, Spearman's correlation coefficient between the SWAP-Score and CIFAR-10 validation accuracy for DARTS CNNs is 0.93
I tested it three times and the results were almost the same every time.
 (Spearman's Correlation Coefficient: 0.5966954170713796, 
 Spearman's Correlation Coefficient: 0.5962400067831423,
 Spearman's Correlation Coefficient: 0.5964299289647295)

**Comment by pym1024** [comment]
Hi there,

It's due to a mistake in the line 29 of swap.py. We did some explorations with _nn.functional.threshold_, but forgot to switch it back to _torch.sign_ when committing to the Github. We've fixed it, you can pull the latest version or fix it by yourself. 
Thanks for letting us know, and feel free to leave comments if you've any further questions.

Cheers
